terraform/snowflake/modules/policies/main.tf

```diff
@@ -0,0 +1,145 @@
+######################################
+#            Terraform               #
+######################################
+# This module enforces Snowflake security by creating a POLICIES database,
+# defining strong default password/authentication policies for different user types,
+# setting Okta-only auth as the default (when enabled), and provisioning a Sentinel
+# legacy service user with the required role grants.
+############################
+#         Providers        #
+############################
+
+terraform {
+  required_providers {
+    snowflake = {
+      source  = "Snowflake-Labs/snowflake"
+      version = "~> 1.0"
+      configuration_aliases = [
+        snowflake.accountadmin,
+        snowflake.securityadmin,
+        snowflake.sysadmin,
+        snowflake.useradmin,
+      ]
+    }
+  }
+  required_version = ">= 1.0"
+}
+
+// Create the POLICIES database to store the password policy
+resource "snowflake_database" "policies" {
+  name = var.policies_database_name // name of the policies database from variable
```

Comment:
This will potentially run into some problems if it is applied after the docs [here](https://cagov.github.io/data-infrastructure/setup/snowflake-setup/#apply-a-session-policy). There's a few places in the docs where we should do some updating (can be in a follow-up)